checkov/terraform/context_parsers/registry.py

```diff
@@ -26,24 +31,24 @@ def reset_definitions_context(self) -> None:
 
     def enrich_definitions_context(
         self, definitions: Tuple[str, Dict[str, List[Dict[str, Any]]]], collect_skip_comments: bool = True
-    ) -> Dict[str, Dict[str, Dict[str, Any]]]:
+    ) -> Dict[str | TFDefinitionKey, Dict[str, Dict[str, Any]]]:
         supported_definitions = [parser_type for parser_type in self.context_parsers.keys()]
         (tf_definition_key, definition_blocks_types) = definitions
+        enable_definition_key = strtobool(os.getenv('ENABLE_DEFINITION_KEY', 'False'))
```

Comment:
I might be missing something, but I'm not familiar with `ENABLE_DEFINITION_KEY` env variable.
The one I know of is `CHECKOV_NEW_TF_PARSER`. where is this one defined?

Reply:
it's a new one. and its will be modified from the platform under FF

Reply:
cool :)